mypy/stubtest.py

```diff
@@ -569,7 +569,7 @@ def get_type(arg: Any) -> str | None:
 
         def has_default(arg: Any) -> bool:
             if isinstance(arg, inspect.Parameter):
-                return arg.default != inspect.Parameter.empty
+                return bool(arg.default != inspect.Parameter.empty)
```

Comment:
I am just curious: why `!=` here does not return `bool`?

Reply:
I think because of this! https://github.com/python/typeshed/blob/8715951fca094fa801c46f860e9aba1b5018874a/stdlib/inspect.pyi#L369